### ISSUE TITLE
Package hyperref Warning: The PDF version number could not be set,(hyperref)

### ISSUE THREAD

**Issue by grosdode** [tudace/tuda_latex_templates#424]
Hello,
if I go to https://sharelatex.tu-darmstadt.de/ and choose _TUDaPhD-3.29_ template I get the following waring 
```
Package hyperref Warning: The PDF version number could not be set,
(hyperref)                because some PDF objects are already written:
(hyperref)                  3 PDF objects
(hyperref)                The version should be set as early as possible:
(hyperref)                  񑙱=1, 񑙲=4\relax
(hyperref)                  \pdfobjcompresslevel=0\relax
(hyperref)                Or \RequirePackage{pdf14} can be used
(hyperref)                before \documentclass as shortcut.
```

**Comment by TeXhackse** [comment]
This is know and shouldn't be an issue at all. The reasons for this are a bit complicated to explain. I will find to find some free time to do so. But you should get a valid PDF out of it without any issues.

Removing that working would require changes in the template syntax. We will do that asap, but this requires a lot of documentation before we can have that within the release.

**Comment by grosdode** [comment]
Sounds good, thank you. 
It was no error, but I try to get at best 0 warnings because a lot of warnings are also problematic/helpful (double defined labels … ). Already starting with a long one where the cause and effect is not clear did not seem good to me.

**Comment by TeXhackse** [comment]
I will keep this issue open, because you are right. People might be confused by this.

**Comment by evamvid** [comment]
Thanks for the quick reply! I tried it a few times with no luck. But I did a little more digging, seems like it doesn't have to do directly with hyperref -- I think that the problem is that I'm not calling \maketitle (my institute has its own titlepage, so I'm using a titlepage environment with the authoraftertitle package to build my own), so the \ptxcd_pass_TitleData function never gets called, meaning the Metadata never gets written into the jobname.xmpdata file for pdfx. Is there a way to manually call that function from the document? I tried to redefine the \maketitle macro in the document to just do that, but no luck.
